src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -254,49 +240,40 @@ def forward(self, x):
 
 class BloomScaledSoftmax(nn.Module):
     """
-    fused operation: scaling + mask + softmax
-
     Args:
+    fused operation: scaling + mask + softmax
         input_in_fp16 (`bool`, *required*):
             flag to indicate if input in fp16 data format.
         input_in_bf16 (`bool`, *required*):
             flag to indicate if input in bf16 data format.
         scaled_masked_softmax_fusion (`bool`, *required*):
             flag to indicate user want to use softmax fusion
-        mask_func (`function`, *required*):
-            mask function to be applied.
         softmax_in_fp32 (`bool`, *required*):
             if true, softmax in performed at fp32 precision.
         scale (`float`, *required*):
             scaling factor used in input tensor scaling.
     """
 
-    def __init__(self, scaled_masked_softmax_fusion, mask_func, softmax_in_fp32, scale):
+    def __init__(self, scaled_masked_softmax_fusion, softmax_in_fp32, scale):
         super().__init__()
         self.scaled_masked_softmax_fusion = scaled_masked_softmax_fusion
```

Comment:
```suggestion
```

Reply:
I don't think this is ever used no? Can we remove it everywhere in this case (also from the docstrings)